.github/workflows/run-tests.yml

```diff
@@ -20,10 +20,24 @@ on:
 
 jobs:
   run-tests:
-    name: Run tests job
+    name: Run tests job matrix
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
```

Comment:
praise: :heart_eyes: :heart_eyes: :heart_eyes:  I did not know we could do this.  And I really want it.  Thank you very much for opening this draft.